### ISSUE TITLE
parameter setting to train on ImageNet from scratch  

### ISSUE THREAD

**Issue by zhiyuanli1992** [asyml/vision_transformer#8]
Thank you for your great work. 
Would you please post your training scripts to train the model on ImageNet. For example the learning rate, weight decay, training_step and etc? 
Thank you very much!

**Comment by DevilYangS** [comment]
+1

**Comment by cjsg** [comment]
Did you actually try to train your model from scratch, or did you always use the pre-trained weights?

In particular, in the README, you write 
> Similar results as in original implementation are achieved.

Do you mean, when training from scratch, or when importing pre-trained weights and fine-tuning?

**Comment by Hhhhhhao** [comment]
The cifar results are fine-tuned from the pretrained weights, not training from scratch

**Comment by zhiyuanli1992** [comment]
> ar results are fine-tuned from the pretrained weights, not training from scratch

Hi, can I have the training script to train on ImageNet from scratch? Thanks!
